apps/retrospective/services/sprint_sync.go

```diff
@@ -209,15 +206,22 @@ func (service SprintService) SyncSprintMemberData(sprintMemberID string) (err er
 		service.SetSyncFailed(sprint.ID)
 		return err
 	}
-
-	var timeTrackerTaskKeys []string
-	var timeLogs []timeTrackerSerializers.TimeLog
-	timeTrackerTaskKeys, timeLogs, err = service.GetSprintMemberTimeTrackerData(sprintMember, sprint)
+	timeProviderConfig := sprintMember.Member.TimeProviderConfig
+	if sprint.Retrospective.TimeProviderName == constants.JIRA {
```

Comment:
Why do we need constant, when we can use the tracker constants?

Reply:
done.